src/codeflare_sdk/ray/cluster/build_ray_cluster.py

```diff
@@ -213,11 +213,10 @@ def get_labels(cluster: "codeflare_sdk.ray.cluster.Cluster"):
     return labels
 
 
-def get_nb_annotations():
+def with_nb_annotations(annotations):
```

Comment:
nit: explicitly add the type:
```suggestion
def with_nb_annotations(annotations: dict):
```